media-sound/cantata/cantata-2.0.0.ebuild

```diff
@@ -0,0 +1,146 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+KDE_REQUIRED="optional"
+PLOCALES="cs de en_GB es hu ko pl ru zh_CN"
+inherit cmake-utils gnome2-utils l10n qmake-utils xdg
+
+DESCRIPTION="Featureful and configurable Qt client for the music player daemon (MPD)"
+HOMEPAGE="https://github.com/CDrummond/cantata"
+SRC_URI="https://github.com/CDrummond/cantata/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="4"
+KEYWORDS="~amd64 ~x86"
+IUSE="cdda cddb http-server kde mtp musicbrainz qt5 replaygain taglib udisks"
+REQUIRED_USE="
+	cdda? ( udisks || ( cddb musicbrainz ) )
+	cddb? ( cdda taglib )
+	mtp? ( taglib udisks )
+	musicbrainz? ( cdda taglib )
+	qt5? ( !kde )
+	replaygain? ( taglib )
+"
+
+RDEPEND="
+	sys-libs/zlib
+	x11-libs/libX11
+	dev-db/sqlite:3
+	kde-frameworks/oxygen-icons
```

Comment:
Please add an alternative with kde-frameworks/breeze-icons (where breeze icons should be the first choice as oxygen-icons are deprecated).